.mergify.yml

```diff
@@ -1,18 +1,62 @@
 # See https://doc.mergify.io
 
 pull_request_rules:
-- name: auto_merge
-  conditions: 
-    - -title~=(WIP|wip)
-    - -merged
-    - -closed
-    - -locked
-    - -label=blocked
-    - -label=do-not-merge
-    - "#approved-reviews-by>=1"
-    - "#changes-requested-reviews-by=0"
-    - status-success=AWS CodeBuild us-east-1 (aws-cdk-build)
-  actions:
-    merge:
-      method: squash
-      strict: smart
+  - name: automatic merge non-sensitive
+    actions:
+      comment:
+        message: Thank you for contributing! Your pull request is now being automatically merged.
+      merge:
+        strict: smart
+        method: squash
+        strict_method: squash
+      delete_head_brancH: {}
+      dismiss_reviews: {}
+    conditions:
+      - -title~=(WIP|wip)
+      - -label~=^(blocked|do-not-merge)
+      - -merged
+      - -closed
+      - "#approved-reviews-by>=1"
+      - -approved-reviews-by~=author
+      - "#changes-requested-reviews-by=0"
+      - "#commented-reviews-by=0"
+      - status-success=AWS CodeBuild us-east-1 (aws-cdk-build)
+      - status-success=Semantic Pull Request
```

Comment:
I am wondering, is it possible to issue a comment on the PR in case PR build fails?

Reply:
[Fixed!](https://github.com/aws/aws-cdk/commit/8bcab058aae95f86e4689acc51b498799bf06e90)